tests/pytests/unit/modules/test_schedule.py

```diff
@@ -93,39 +95,41 @@ def test_delete(sock_dir, job1):
     """
     Test if it delete a job from the minion's schedule.
     """
-    with patch.dict(schedule.__opts__, {"schedule": {}, "sock_dir": sock_dir}):
-        mock = MagicMock(return_value=True)
-        with patch.dict(schedule.__salt__, {"event.fire": mock}):
-            _ret_value = {"complete": True, "schedule": {}}
-            with patch.object(SaltEvent, "get_event", return_value=_ret_value):
-                assert schedule.delete("job1") == {
-                    "comment": "Job job1 does not exist.",
-                    "changes": {},
-                    "result": False,
-                }
+    with patch("os.makedirs", MagicMock(return_value=True)):
+        with patch.dict(schedule.__opts__, {"schedule": {}, "sock_dir": sock_dir}):
+            mock = MagicMock(return_value=True)
+            with patch.dict(schedule.__salt__, {"event.fire": mock}):
+                _ret_value = {"complete": True, "schedule": {}}
+                with patch.object(SaltEvent, "get_event", return_value=_ret_value):
+                    assert schedule.delete("job1") == {
+                        "comment": "Job job1 does not exist.",
+                        "changes": {},
+                        "result": False,
+                    }
 
     _schedule_data = {"job1": job1}
     comm = "Deleted Job job1 from schedule."
     changes = {"job1": "removed"}
-    schedule_config_file = schedule._get_schedule_config_file()
-    with patch.dict(
-        schedule.__opts__, {"schedule": {"job1": "salt"}, "sock_dir": sock_dir}
-    ):
-        with patch("salt.utils.files.fopen", mock_open(read_data="")) as fopen_mock:
-            with patch.object(
-                schedule, "list_", MagicMock(return_value=_schedule_data)
-            ):
-                assert schedule.delete("job1", offline="True") == {
-                    "comment": comm,
-                    "changes": changes,
-                    "result": True,
-                }
+    with patch("os.makedirs", MagicMock(return_value=True)):
+        schedule_config_file = schedule._get_schedule_config_file()
+        with patch.dict(
+            schedule.__opts__, {"schedule": {"job1": "salt"}, "sock_dir": sock_dir}
+        ):
+            with patch("salt.utils.files.fopen", mock_open(read_data="")) as fopen_mock:
+                with patch.object(
+                    schedule, "list_", MagicMock(return_value=_schedule_data)
+                ):
```

Comment:
**suggestion (non-blocking)** you can guess what I'm going to suggest here, huh? :joy: